lib/cog/aggregate_command.rb

```diff
@@ -0,0 +1,57 @@
+require 'cog/exceptions'
+
+class Cog
+  class AggregateCommand < Cog::Command
+
+    def initialize
+      subcommands = self.class.const_get("SUBCOMMANDS")
+
+      unless subcommand = request.args.shift
+        no_subcommand_error
+        response.abort
+        return
```

Comment:
You can see how I'm handling it in the cfn command here https://github.com/cogcmd/aws-cfn/pull/9/files#diff-a60595cd4eae1e61f357520b7acaf64cR132